Trigger/src/ReadTriggerInfo_module.cc

```diff
@@ -366,6 +325,11 @@ namespace mu2e {
     Hist._h2DTrigInfo[0] = trigInfoDir.make<TH2F>("h2DTrigInfo_map_all" , "Trigger correlation map from all filters"   , (_nMaxTrig+2), -0.5, (_nMaxTrig+1.5), (_nMaxTrig+2), -0.5, (_nMaxTrig+1.5));
     Hist._h2DTrigInfo[1] = trigInfoDir.make<TH2F>("h2DTrigInfo_map"     , "Trigger correlation map"                    , (_nMaxTrig+2), -0.5, (_nMaxTrig+1.5), (_nMaxTrig+2), -0.5, (_nMaxTrig+1.5));
 
+    //add a for loop for creating histograms like
+    for(int i=0; i<30; ++i){
```

Comment:
Don't use literal numbers.  Is 30 already defined in an another variable?  If not please do:

```
constexpr int variable_with_a_descriptive_name{30};
for ( int i=0; i<variable_with_a_descriptive_name; ++i){
  // body of the loop
 }
```
Alternatively, use a simple name but with a descriptive comment.